### PR TITLE
Hwas and BrokerLock added to doxygen

### PR DESCRIPTION
**Pull request by jrachucki-n7s**

### CODE REVIEW

File: Doxyfile [hunk 1]
```diff
@@ -833,6 +833,8 @@ WARN_LOGFILE           =
 
 INPUT                  = src/Hal \
                          src/Init \
+                         src/hwas \
```
Comment (Glamhoth) [inline Doxyfile:836]: Directory name should be capitalized, could you fix it in one go?

**Review by Glamhoth — CHANGES_REQUESTED** [review]

File: Doxyfile (hunk 1 above)
Comment (jrachucki-n7s) [reply Doxyfile:836]: I am not sure. This is generated code, I don't know if changing the folder name won't affect the build of the code that depends on hwas. @Lurkerpas can we change the name of this folder.

Comment (Lurkerpas) [reply Doxyfile:836]: @Glamhoth -> yes, this is generated code. Let's keep it as is for now.

Comment (Glamhoth) [reply Doxyfile:836]: ack

**Review by Glamhoth — APPROVED** [review]

**Pull request merged**